package.json

```diff
@@ -86,5 +91,8 @@
   "prettier": {
     "semi": false,
     "singleQuote": true
+  },
+  "lint-staged": {
+    "*.{ts,tsx,js}": "eslint --cache --fix"
```

Comment:
I never worked with `lint-staged` and am a bit confused. There is this line here and a config file with different content. Do we need both? And can we replace the current "lint" command we have with this? CI is running `npm run lint` which only does the `eslint` part, not the typescript checks that are in the config file.

Reply:
I started adding it to projects recently. Mostly because the CI. Works with git hooks setup after doing `npm install`. Basically added it because most of the time the CI reported lint errors that I should have fixed before pushing... and thats the only reason to have it. 

Sure I'll adjust the command.

If you want I can move this to a separate PR.

Reply:
I am okay with improving it here. And yes, it's a nice addition to the workflow. I like the idea.

Reply:
This still needs to be updated, right?

Reply:
Ah yes! This is actually now at `lint-stagged.config.js`. Will remove it from here